### ISSUE TITLE
Update so that db does not wipe on server restart

### ISSUE THREAD

**Issue by timothycobrien** [vu-cs3892-21s/Group7#10]
Use volume to preserve data within docker container, just conditionally create tables if necessary

**Comment by EvanW1999** [comment]
Yeah I think this is what you're supposed to use: https://flask-migrate.readthedocs.io/en/latest/
